client/templates/register/register.scss

```diff
@@ -1,12 +1,3 @@
-.register {
```

Comment:
I believe we want to keep all content of this .scss within nested within .register. Gal added a package that applies styles to the body [that have the same name as the route you're currently on.](https://github.com/brianchitester/facequalm/blob/dev/lib/routes.js#L33)